src/services/review/RepoCheckerService.ts

```diff
@@ -426,11 +426,10 @@ export default class RepoCheckerService {
                 return errAsync("No permalink found in front matter")
               }
               const normalisedPermalink = this.normalisePermalink(filePermalink)
-
               // we are only parsing out the front matter
               const fileContent = file.split("---")?.slice(2).join("---")
               if (!fileContent) {
-                return errAsync("No file content found")
+                return errAsync(`No page content for ${fileName}`) // it is ok to have no page content
```

Comment:
ok w/ the change - the alarm si doing a string search, thus necessitating this change?